iam/sc-enduser-iam.yaml

```diff
@@ -22,6 +22,28 @@ Resources:
             Action:
               - 'sts:AssumeRole'
               - 'sts:TagSession'
+  SCExternalEnduserRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: ServiceCatalogExternalEndusers
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AWSServiceCatalogEndUserFullAccess
+        - !Ref SCEnduserExpandedPolicy
+        - !Ref CFNDenyGetTemplateSummaryPolicy
+        - !Ref SCEnduserRemoteAccessPolicy
+      Path: /
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS: "*"
+            Condition:
+              StringLike:
+                aws:PrincipalArn: !Sub 'arn:aws:iam::${AWS::AccountId}:role/synapse-login-scipool*-InstanceRole-*'
+            Action:
+              - 'sts:AssumeRole'
+              - 'sts:TagSession'
```

Comment:
Why is there a separate end user role? Isn't this going to require that we change login app (a lot)? We were planning to use  ABAC with session tags to control portfolio access.

Reply:
it's because portfolio access is done thru roles and we need another role to associate external portfolios to, https://github.com/Sage-Bionetworks/service-catalog-library/blob/master/ec2/sc-portfolio-ec2.yaml#L26

So there is no ABAC usage setup yet. Do you have a suggestion as to what should be done now?  Setup  ABAC  before creating external portfolio? Setup ABAC as a follow on PR?  I'm open to any suggestion you want to propose.